zoo-css-challenge/style.css

```
@@ -64,8 +64,17 @@ button {
   background-color: #372772;
   border: 2px solid black;
   color: #fff;
+  font-size: 17px;
+  transition: 0.3s;
 }
 
+/* change colour when you hover*/
+.button:hover {background-color: #3e8e41}
```

```suggestion
.button:focus, .button:hover {background-color: #3e8e41}
```
We should also add `:focus` so keyboard users who use the `Tab` key to move to buttons will also see the changed background colour